apollo-backend/src/main/java/io/logz/apollo/controllers/BlockerDefinitionController.java

```diff
@@ -98,18 +99,23 @@ public void addBlockerDefinition(String name, String environmentId, String servi
 
         Integer environmentIdParsed = null;
         Integer serviceIdParsed = null;
+        Integer stackIdParsed = null;
 
         if (environmentId != null && !environmentId.equals("null"))
             environmentIdParsed = Integer.parseInt(environmentId);
 
         if (serviceId != null && !serviceId.equals("null"))
             serviceIdParsed = Integer.parseInt(serviceId);
 
+        if (stackId != null && !stackId.equals("null"))
+            stackIdParsed = Integer.parseInt(stackId);
+
         BlockerDefinition blockerDefinition = new BlockerDefinition();
 
         blockerDefinition.setName(name);
         blockerDefinition.setEnvironmentId(environmentIdParsed);
         blockerDefinition.setServiceId(serviceIdParsed);
+        blockerDefinition.setStackId(stackIdParsed);
```

Comment:
Add verification that a blocker  can either   contain a stack, or (environment and/or service).
if blocker.stackId != null -> if blocker.environment != null or blocker.service != null then return 400